plugins/extractors/tableau/tableau.go

```diff
@@ -22,7 +22,6 @@ var summary string
 var sampleConfig = `
 host: https://server.tableau.com
 version: "3.12"
-identifier: my-tableau
```

Comment:
What is this used for?

Reply:
In this meteor PR, `identifier` field was added in extractors for URN generation
https://github.com/odpf/meteor/pull/354/files
Later there was an updation to generate URN in a different manner in this PR, deprecating the identifier field altogether.
https://github.com/odpf/meteor/pull/387/files